contracts/test/integration/vip/vip8.sol

```diff
@@ -0,0 +1,116 @@
+//SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+import {TimelockSimulation} from "../utils/TimelockSimulation.sol";
+import {MainnetAddresses} from "../fixtures/MainnetAddresses.sol";
+import {ArbitrumAddresses} from "../fixtures/ArbitrumAddresses.sol";
+import {DSTest} from "./../../unit/utils/DSTest.sol";
+import {Core} from "../../../core/Core.sol";
+import {Vm} from "./../../unit/utils/Vm.sol";
+import {IVIP} from "./IVIP.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {PCVGuardian} from "../../../pcv/PCVGuardian.sol";
+import {PegStabilityModule} from "../../../peg/PegStabilityModule.sol";
+
+contract vip8 is DSTest, IVIP {
+    using SafeERC20 for IERC20;
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+
+    function getMainnetProposal()
```

Comment:
We need to add an integration test here that runs vip8 first, and then does a swap with the VOLT DAI PSM after the maker router swap completes.

Reply:
Integration test should be in a separate file, and could extend the test run in VIP7 with the caveat that it must run VIP8 as a timelock simulated action.